site/downloads.markdown

```diff
@@ -20,13 +27,16 @@ This page describes the installation of the Haskell toolchain, which consists of
 1. Install GHC, cabal-install and haskell-language-server via [GHCup](https://www.haskell.org/ghcup/)
 2. To install Stack, follow the [Stack installation guide](https://docs.haskellstack.org/en/stable/install_and_upgrade/)
 
+<br/>
+
 * * *
 
+
 ### Via native OS package manager
 
-Alternatively, many operating systems provide GHC, cabal and Stack through their native package manager.  The packages are often out-of-date but if you prefer to use this method of installation then you will find useful links below.
+Alternatively, many operating systems provide GHC, cabal and Stack through their native package manager.  The packages are often out-of-date and we don't recommend this method of installation, but if you prefer to use this method then you will find useful links below.
 
```

Comment:
Can you please drop this part? The current wording is part of a compromise to satisfy a segment of the community which insists that operating system packages are the one true way to install Haskell (or anything). The current wording is literally true: they are often out-of-date. The proposed wording suggests that the Haskell.org committee has made a value judgement about what is or is not to be recommended. I would prefer to avoid that. Rather the relative order in which we present the various options should implicitly convey that information.

Reply:
Sure, done!